natlas-agent/natlas/utils.py

```diff
@@ -14,9 +14,9 @@
 def validate_target(target, config):
 	try:
 		iptarget = ipaddress.ip_address(target)
-		if iptarget.is_private and not config.scan_local:
-			utillogger.error("We're not configured to scan local addresses!")
-			return False
+		# if iptarget.is_private and not config.scan_local:
+		# 	utillogger.error("We're not configured to scan local addresses!")
+		# 	return False
```

Comment:
What do you think about me making this a Environment variable

Reply:
Making what an environment variable? config.scan_local? It is an environment variable, it's just loaded from config.py (where all environment variables should be loaded from)

Reply:
There we go. Now I need to fix the elasticsearch url too.